heracles/dices/shrinkage.py

```diff
@@ -103,76 +94,75 @@ def shrinkage_factor(cls1, target):
     return lambda_star
 
 
-def gaussian_covariance(Cls):
-    """
-    Computes Gaussian estimate of the target matrix.
-    input:
-        Cls: power spectra
-    returns:
-        T: target matrix
-    """
-    # Add bias to Cls
-    b = bias(Cls)
-    Cls = add_to_Cls(Cls, b)
-    # Separate Cls into Cls
-    _Cls = _fields2components(Cls)
-    # Compute Gaussian covariance
+def gaussian_covariance(cls):
+    b = bias(cls)
+    cls = add_to_Cls(cls, b)
     cov = {}
-    for key1, key2 in itertools.combinations_with_replacement(Cls, 2):
-        # covariance key
+    for key1, key2 in itertools.combinations_with_replacement(cls.keys(), 2):
         a1, b1, i1, j1 = key1
         a2, b2, i2, j2 = key2
         covkey = (a1, b1, a2, b2, i1, j1, i2, j2)
         # get reference results
-        cl1 = Cls[key1]
-        cl2 = Cls[key2]
+        cl1 = cls[key1]
+        cl2 = cls[key2]
```

Comment:
We only need these for their shapes, so expand the spin-0 dimensions with the new helper function.

```suggestion
        cl1 = expand_spin0_dims(cls[key1])
        cl2 = expand_spin0_dims(cls[key2])
```